code/datums/recipe.dm

```diff
@@ -55,6 +55,30 @@
 		return -1
 	return .
 
+/**
+ * Similarly to the function above, this checks for reagents, except instead of being passed a reagent holder, we're passed
+ * [reagent_id] = amount as num.
+ * Returns 1 if we have the precise amount thats requested.
+ * Returns 0 if we do not have enough.
+ * Returns -1 if we have MORE than requested.
```

Comment:
It would be nice to make these variables defines, rather than just magic numbers.